.github/assignment.yml

```diff
@@ -0,0 +1,49 @@
+---
+- rule:
+    path: "specification/**/Microsoft.Network/**"
```

Comment:
Will we have this kind of define in a file like codeowners.  e.g reviewers, right?

Reply:
I think this format is more flexible